facebook/OpenBIC
# fby35: gl: Enable ADC's deglitch and set upper bound

# Description:
    - Enable ADC's deglitch and set upper bound.
        - ADC sensors:
            - MB_ADC_P12V_STBY_VOLT_V
            - MB_ADC_P3V_BAT_VOLT_V
            - MB_ADC_P3V3_STBY_VOLT_V
            - MB_ADC_P1V8_STBY_VOLT_V
            - MB_ADC_P1V05_STBY_VOLT_V
            - MB_ADC_P5V_STBY_VOLT_V
            - MB_ADC_P12V_DIMM_VOLT_V
            - MB_ADC_P1V2_STBY_VOLT_V
            - MB_ADC_P3V3_M2_VOLT_V

# Motivation:
    - Set the maximum readable value for the ADC. If the ADC sensor exceeds it, the BIC will read the ADC again.

# Test Plan:
    1. build code: Pass
    2. Check ADC sensor: Pass
        root@bmc-oob:~# sensor-util slot3 | grep "ADC"
        MB_ADC_P12V_STBY_VOLT_V      (0x17) :   12.59 Volts | (ok)
        MB_ADC_P3V_BAT_VOLT_V        (0x18) :    3.23 Volts | (ok)
        MB_ADC_P3V3_STBY_VOLT_V      (0x19) :    3.38 Volts | (ok)
        MB_ADC_P1V8_STBY_VOLT_V      (0x1A) :    1.81 Volts | (ok)
        MB_ADC_P1V05_STBY_VOLT_V     (0x1B) :    1.07 Volts | (ok)
        MB_ADC_VNN_VOLT_V            (0x23) :    1.00 Volts | (ok)
        MB_ADC_P5V_STBY_VOLT_V       (0x24) :    5.01 Volts | (ok)
        MB_ADC_P12V_DIMM_VOLT_V      (0x25) :   12.45 Volts | (ok)
        MB_ADC_P1V2_STBY_VOLT_V      (0x26) :    1.20 Volts | (ok)
        MB_ADC_P3V3_M2_VOLT_V        (0x27) :    3.33 Volts | (ok)
